File: chapter05/functions_1.exs
list_concat = fn(xs, ys) -> xs ++ ys end
list_concat.([:a, :b], [:c, :d])

sum = fn(x, y, z) -> x + y + z end
sum.(1, 2, 3)

pair_tuple_to_list = fn({x, y}) -> [x, y] end
pair_tuple_to_list.({1234, 5678})

File: chapter05/functions_1.clj
(def list-concat
  (fn [xs ys] (concat xs ys)))
(list-concat [:a :b] [:c :d])

(def sum
  (fn [x y z] (+ x y z)))
(sum 1 2 3)

(def pair-tuple->list
  (fn [[x y]] (list x y)))
(pair-tuple->list [1234 5678])
